### PR TITLE
Convert admin RABL views to Jbuilder

### PR DESCRIPTION
**Pull request by jhawthorn**

### CODE REVIEW

File: backend/solidus_backend.gemspec [hunk 1]
```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'jquery-ui-rails', '~> 5.0.0'
   s.add_dependency 'font-awesome-rails', '~> 4.0'
   s.add_dependency 'kaminari', '>= 0.17', '< 2.0'
+  s.add_dependency 'jbuilder'
```
Comment (tvdeyen) [inline backend/solidus_backend.gemspec]: We should set version numbers

Comment (jhawthorn) [inline backend/solidus_backend.gemspec]: :+1: done

**Comment by cbrunsdon** [comment]
When you see anything that looks like this: `my_instance.()` it calls the `call` method on that object, if its an object, or will invoke a proc if its a proc.

**Comment by jhawthorn** [comment]
> When you see anything that looks like this: `my_instance.()` it calls the `call` method on that object, if its an object, or will invoke a proc if its a proc.

Yeah, JBuilder uses this (maybe too) cleverly as a shorthand for multiple attributes. I replaced a few spots where it didn't make sense to use this over explicitly declaring the attributes on separate lines.

**Pull request merged**